package.json

```diff
@@ -25,52 +25,51 @@
   },
   "homepage": "https://github.com/uc-cdis/gen3-ui-component#readme",
   "dependencies": {
-    "@storybook/addon-actions": "^3.4.10",
-    "@storybook/react": "^3.4.8",
-    "babel-polyfill": "^6.26.0",
+    "@babel/core": "^7.3.4",
```

Comment:
I think this one should be in `devDependencies`, not `dependencies`? also do other babel packages...